packages/machine-extractor/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "outDir": "out",
     "rootDir": "src",
     "noEmit": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "exactOptionalPropertyTypes": true
```

Comment:
Is this in general a good thing to enable? Wouldn't mind having this in the studio

Reply:
I think so. However, this would require a lot more work than in this repo - since the Studio is a much bigger project and it also integrates with way more packages.